Makefile

```
@@ -305,41 +203,46 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
-### install_cert_manager: install Cert Mananger v1.0.4 on the cluster
-install_cert_manager:
-	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
 
-_operator_sdk:
-	@{ \
-		if ! command -v operator-sdk &> /dev/null; then \
-			echo 'operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but it is not installed' ;\
-			exit 1 ;\
-		else \
-			SDK_VER=$$(operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2) && \
-			if [ "$${SDK_VER}" != $(OPERATOR_SDK_VERSION) ]; then \
-				echo "WARN: operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but $${SDK_VER} found" \
-				echo "WARN: Please use the recommended operator-sdk if you face any issue" \
-			fi \
-		fi \
-	}
-ifneq ($(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2),$(OPERATOR_SDK_VERSION))
-	@echo 'WARN: operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but $(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2) found.'
-	@echo 'WARN: Please use the recommended operator-sdk if you face any issue.'
-endif
-
-_check_cert_manager:
-ifeq ($(PLATFORM),kubernetes)
-	if ! ${K8S_CLI} api-versions | grep -q '^cert-manager.io/v1$$' ; then \
-		echo "Cert-manager is required for deploying on Kubernetes. See 'make install_cert_manager'" ;\
-		exit 1 ;\
-	fi
-endif
+### compile-devworkspace-controller: Compiles the devworkspace-controller binary
+.PHONY: compile-devworkspace-controller
+compile-devworkspace-controller:
+	export ARCH="$(shell uname -m)";  \
+	if [[ "$(ARCH)" == "x86_64" ]]; then  \
+		export ARCH="amd64";  \
+	elif [[ "$(ARCH)" == "aarch64" ]]; then  \
+		export ARCH="arm64";  \
+	fi  \
```

Missing a semicolon here (and in `compile-webhook-server`):
```suggestion
	fi  ;\
```

```
Step 8/21 : RUN make compile-devworkspace-controller
 ---> Running in cc6ffbdb3a97
bash: -c: line 6: syntax error near unexpected token `CGO_ENABLED=0'
make: *** [Makefile:210: compile-devworkspace-controller] Error 2
The command '/bin/sh -c make compile-devworkspace-controller' returned a non-zero code: 2
make: *** [Makefile:178: docker-build] Error 2
```

---

yikes! good catch, added it :-)